synonyms/custom_admin.txt

```diff
@@ -23,3 +23,4 @@
 #   foo => foo bar, baz
 #
 # =============================================================================
+GB => UK
```

Comment:
This should be "GB,UK" (with a comma) as per `synonyms/country_a2/country_a2.txt`.